### PR TITLE
FIX-#1144: Fix `read_parquet` for working with HDFS

### LINKED ISSUE #1144: modin.pandas.read_parquet() files from HDFS working unexpectedly as compared to pandas.read_parquet()
### System information
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Ubuntu 18.04
- **Modin version** (`modin.__version__`): 0.7.2
- **Python version**: 3.6.6

### Describe the problem

^[[Aos.environ['JAVA_HOME'] = '/usr/jdk64/jdk1.8.0_112/jre'
KeyboardInterrupt
>>> os.environ["MODIN_ENGINE"] = "dask"
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
NameError: name 'os' is not defined
>>> import os
>>> os.environ["MODIN_ENGINE"] = "dask"
>>> import modin.pandas as pd1
>>> import pandas as pd
>>> import numpy as np
>>> import pyarrow
i>>> import os
s>>> os.environ['ARROW_LIBHDFS_DIR'] = '/usr/hdp/3.1.4.0-315/usr/lib'
>>> os.environ['JAVA_HOME'] = '/usr/jdk64/jdk1.8.0_112/jre'
>>>
>>>
>>> df = pd.read_parquet('hdfs://ip:port/abc.parquet’)
2020-03-17 00:35:29,552 WARN util.NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
>>> df1 = pd1.read_parquet('hdfs://ip:port/abc.parquet')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/username/.local/lib/python3.6/site-packages/modin/pandas/io.py", line 27, in read_parquet
    path=path, columns=columns, engine=engine, **kwargs
  File "/home/username/.local/lib/python3.6/site-packages/modin/data_management/factories.py", line 44, in read_parquet
    return cls._determine_engine()._read_parquet(**kwargs)
  File "/home/username/.local/lib/python3.6/site-packages/modin/data_management/factories.py", line 48, in _read_parquet
    return cls.io_cls.read_parquet(**kwargs)
  File "/home/username/.local/lib/python3.6/site-packages/modin/engines/ray/generic/io.py", line 128, in read_parquet
    pf = ParquetFile(path)
  File "/home/username/.local/lib/python3.6/site-packages/ray/pyarrow_files/pyarrow/parquet.py", line 132, in __init__
    self.reader.open(source, use_memory_map=memory_map, metadata=metadata)
  File "pyarrow/_parquet.pyx", line 1002, in pyarrow._parquet.ParquetReader.open
  File "pyarrow/io.pxi", line 1323, in pyarrow.lib.get_reader
  File "pyarrow/io.pxi", line 1312, in pyarrow.lib._get_native_file
  File "pyarrow/io.pxi", line 780, in pyarrow.lib.memory_map
  File "pyarrow/io.pxi", line 743, in pyarrow.lib.MemoryMappedFile._open
  File "pyarrow/error.pxi", line 78, in pyarrow.lib.check_status
pyarrow.lib.ArrowIOError: Failed to open local file 'hdfs://ip:port/abc.parquet’, error: No such file or directory

### PR DESCRIPTION
**Pull request by prutskov**
Signed-off-by: Alexey Prutskov <alexey.prutskov@intel.com>

## What do these changes do?

- [x] commit message follows format outlined [here](https://modin.readthedocs.io/en/latest/contributing.html)
- [x] passes `flake8 modin`
- [x] passes `black --check modin`
- [x] signed commit with `git commit -s` 
- [x] Resolves #1144  
- [ ] tests added and passing

### CODE REVIEW

**Comment by prutskov** [comment]
@devin-petersohn can you review this, please?
Currently this can be tested locally only due to need an additional tools like hadoop, java. Issue related to CI-testing was created #2121.

File: modin/engines/base/io/column_stores/parquet_reader.py [hunk 1]
```diff
@@ -66,6 +66,17 @@ def _read(cls, path, engine, columns, **kwargs):
                 pd = ParquetDataset(path)
                 meta = pd.metadata
                 column_names = pd.schema.names
+            elif (
+                isinstance(path, str)
+                and "://" in path
+                and not path.startswith(("http://", "https://"))
```
Comment (devin-petersohn) [inline modin/engines/base/io/column_stores/parquet_reader.py:72]: Is http:// or https:// handled by `ParuqetFile` constructor? The documentation did not say: https://arrow.apache.org/docs/python/generated/pyarrow.parquet.ParquetFile.html

Comment (prutskov) [reply modin/engines/base/io/column_stores/parquet_reader.py]: You're right. It was fixed. Now I try to handle `hdfs://` beginning only.

**Review by devin-petersohn — APPROVED** [review]
Thanks @prutskov, LGTM!

**Comment by kocmurat93** [comment]
hi , 
this topic is very important for our projects 
when will you fix this problem   ? 
this topic is open about 1 year :)

thanks ! 

Murat KOÇ

**Pull request merged**